monitoring/bin/deploy_monitoring_cluster.sh

```diff
@@ -240,6 +240,23 @@ enable_pod_token_automount $MON_NS deployment v4m-operator
 log_info "Deploying ServiceMonitors and Prometheus rules"
 log_verbose "Deploying cluster ServiceMonitors"
 
+TRACING_ENABLE="${TRACING_ENABLE:-true}"
+if [ "$TRACING_ENABLE" == "true" ]; then
+  log_info "Tracing enabled..."
+  # # Add the grafana helm chart repo
+  helmRepoAdd grafana https://grafana.github.io/helm-charts
+  log_info "Installing tempo"
+  helm upgrade --install v4m-tempo --set serviceMonitor.enabled=true --set searchEnabled=true -n "$MON_NS" grafana/tempo
+
+  log_verbose "Provisioning Tempo Datasource"
+  if [ "$TLS_ENABLE" == "false" ]; then
+    grafanaTempoDS=grafana-datasource-tempo.yaml
```

Comment:
I see the non-TLS use-case handled by this block but not the TLS-enabled.  Hmmm, I think I see what's going on...you've added the Tempo datasource definition to the TLS-enabled Prometheus datasource files.  I see how that would work but I'm not convinced that the convenience is worth the potential confusion.
Why not define a grafana-datasource-tempo.yaml file in monitoring/tls?  Your code is already written to use a variable ($grafanaTempoDS) to identify the file.  Let's discuss.